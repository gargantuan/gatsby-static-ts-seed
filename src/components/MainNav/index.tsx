import * as React from 'react';
import { Link } from 'gatsby';

export default () => (
  <nav>
    <ul>
      <li><Link to="/about">About</Link></li>
      <li><Link to="/contact">Contact</Link></li>
    </ul>
  </nav>
);
