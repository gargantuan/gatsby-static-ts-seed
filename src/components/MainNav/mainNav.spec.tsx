import * as React from 'react';
import * as renderer from "react-test-renderer"
import MainNav from './index';

describe("Contact", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<MainNav />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
});
