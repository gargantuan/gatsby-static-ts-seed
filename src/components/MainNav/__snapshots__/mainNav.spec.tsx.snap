// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders correctly 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/about"
      >
        About
      </a>
    </li>
    <li>
      <a
        href="/contact"
      >
        Contact
      </a>
    </li>
  </ul>
</nav>
`;
